lib/buffer.js

```diff
@@ -533,6 +594,30 @@ Buffer.prototype.write = function(string, offset, length, encoding) {
   }
 };
 
+Buffer.prototype.asciiWrite = function(string, offset, length) {
+  return this.write(string, offset, length, 'ascii');
+};
+
+Buffer.prototype.base64Write = function(string, offset, length) {
+  return this.write(string, offset, length, 'base64');
+};
+
+Buffer.prototype.binaryWrite = function(string, offset, length) {
+  return this.write(string, offset, length, 'binary');
+};
+
+Buffer.prototype.hexWrite = function(string, offset, length) {
+  return this.write(string, offset, length, 'hex');
+};
+
+Buffer.prototype.ucs2Write = function(string, offset, length) {
+  return this.write(string, offset, length, 'ucs2');
+};
+
+Buffer.prototype.utf8Write = function(string, offset, length) {
+  return this.write(string, offset, length, 'utf8');
```

Comment:
we might as well deprecate these calls if they're just being sent back to `.write()`. (not this PR though, just thinking out loud)